flink-table/flink-table-common/src/main/java/org/apache/flink/table/catalog/CatalogDescriptor.java

```diff
@@ -40,6 +40,9 @@ public class CatalogDescriptor {
     /* The configuration used to discover and construct the catalog. */
     private final Configuration configuration;
 
+    /* Catalog comment. */
+    private final String comment;
```

Comment:
Add `@Nullable` annotation since comment is not mandatory.

Reply:
done